tests/unit/app/endpoints/test_ols.py

```diff
@@ -173,6 +174,20 @@ def test_validate_question_valid_kw(llm_validate_question_mock, _load_config):
     assert llm_validate_question_mock.call_count == 0
 
 
+@patch(
+    "ols.src.query_helpers.question_validator.QuestionValidator.validate_question",
+    side_effect=PromptTooLongError("Prompt length 10000 exceeds LLM"),
+)
+def test_validate_question_too_long_query(llm_validate_question_mock, _load_config):
+    """Check the behaviour of validate_question function with too long query."""
+    conversation_id = suid.get_suid()
+    query = "Tell me about Kubernetes?" * 10000
```

Comment:
is this necessary?  you've already setup a side effect that validate_question is going to raise the exception regardless.

Reply:
not right now

Reply:
please remove it then, because as it is now it's misleading as to what the test is actually doing/testing.

Reply:
just did it :)